pkg/test/framework/components/apiserver/apiserver.go

```diff
@@ -0,0 +1,48 @@
+//  Copyright 2018 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package apiserver
+
+import (
+	"fmt"
+
+	"istio.io/istio/pkg/test/framework/environment"
+	"istio.io/istio/pkg/test/framework/environments/kubernetes"
+	"istio.io/istio/pkg/test/kube"
+)
+
+// InitKube initializes a new API Server component for the kubernetes environment.
+func InitKube(ctx environment.ComponentContext) (interface{}, error) {
```

Comment:
Not something we need to address now, but I still think that the apiserver shouldn't be a top-level component, but rather something exposed by the kube environment.